src/Nowruz/pages/setting/sections/account/account.tsx

```diff
@@ -0,0 +1,186 @@
+import { Button } from 'src/components/atoms/button/button';
+import { Input } from 'src/Nowruz/modules/general/components/input/input';
+import { Modal } from 'src/components/templates/modal/modal';
```

Comment:
Please never import components from V2. We are going to remove components which are out of Nowruz. Modal, Button, etc. should be imported from Nowruz components. Even In a specific situation in which a basic component is not implemented and you are going to use it from V2 ( I doubt something like that happen), firstly you should move it to Nowruz components and then use it